src/Http/RequestTransformer.php

```diff
@@ -41,6 +41,14 @@ public static function toCake(PsrRequest $request)
     {
         $post = $request->getParsedBody();
         $server = $request->getServerParams();
+        $headers = $request->getHeaders();
+
+        foreach ($headers as $k => $value) {
+            $name = sprintf('HTTP_%s', strtoupper(str_replace('-', '_', $k)));
+            if (isset($server[$name])) {
```

Comment:
I'm assuming you want to be able to set headers even if they are not set in the `$_SERVER` vars as well?